tests/e2e/utils/measure.ts

```diff
@@ -0,0 +1,44 @@
+import {profiler} from '@perf-profiler/profiler';
+import {getAverageCpuUsage, getAverageCpuUsagePerProcess, getAverageFPSUsage, getAverageRAMUsage} from '@perf-profiler/reporter';
+import {ThreadNames} from '@perf-profiler/types';
+import type {Measure} from '@perf-profiler/types';
+
+let measures: Measure[] = [];
+let polling = {
+    stop: (): void => {
```

Comment:
When will this version of the method callled?

I can see that the `pulling` is assigned a value inside `start`, but will the `stop` method change to this original definition when the profiler is stopped?

Reply:
You're right, I added missing code in https://github.com/Expensify/App/pull/43482/commits/7b1994bc6745ae6703b322c30b1f6f334dd494cc 🙌